### DISCUSSION TITLE
recreate devices

### DISCUSSION THREAD

**Discussion by jamesleemarsh** [tsightler/ring-mqtt#472] [Support Q&A] [answered]
I was troubleshooting a problem with my devices coming up unresponsive. i found that it was a error with mqtt. befroe i fixed that i deleted the ring addon and also the devices. i have fixed the original problem but now when i reinstall the plugin i cant get it to recreate the devices. the log shows the devices are responding and working. 

is there a way to force the recreation of the devices?

**Comment by tsightler** [comment, marked as answer]
All device discovery data is sent each and every time the addon is started, as well as any time a Home Assistant restart is detected.  Assuming you have the discovery debug options enabled you will see these messages in the log during addon startup.  If the devices aren't appearing in Home Assistant, then there's likely still a problem with MQTT.

**Comment by jamesleemarsh** [comment]
when you right. you right
